pkg/cmd/server/apis/config/helpers.go

```diff
@@ -376,31 +376,48 @@ func IsPasswordAuthenticator(provider IdentityProvider) bool {
 func IsIdentityProviderType(provider runtime.Object) bool {
 	switch provider.(type) {
 	case
-		(*RequestHeaderIdentityProvider),
-		(*BasicAuthPasswordIdentityProvider),
-		(*AllowAllPasswordIdentityProvider),
-		(*DenyAllPasswordIdentityProvider),
-		(*HTPasswdPasswordIdentityProvider),
-		(*LDAPPasswordIdentityProvider),
-		(*KeystonePasswordIdentityProvider),
-		(*OpenIDIdentityProvider),
-		(*GitHubIdentityProvider),
-		(*GitLabIdentityProvider),
-		(*GoogleIdentityProvider):
+		*RequestHeaderIdentityProvider,
+		*BasicAuthPasswordIdentityProvider,
+		*AllowAllPasswordIdentityProvider,
+		*DenyAllPasswordIdentityProvider,
+		*HTPasswdPasswordIdentityProvider,
+		*LDAPPasswordIdentityProvider,
+		*KeystonePasswordIdentityProvider,
+		*OpenIDIdentityProvider,
+		*GitHubIdentityProvider,
+		*GitLabIdentityProvider,
+		*GoogleIdentityProvider:
 
 		return true
 	}
 
 	return false
 }
 
+func IsGroupsIdentityProvider(provider runtime.Object) bool {
+	switch provider.(type) { // TODO determine if this list is reasonable
```

Comment:
list of possible IDPs supporting groups (basically all of them)

Reply:
why do we care to make this list ?

Reply:
It matters less now since I simplified how this is configured.  I mostly wanted to avoid users thinking an IDP supported groups when it does not.